From ad491b9ea142d5a83485a3ddcac8fff28965f100 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 2 May 2014 14:37:46 -0400
Subject: [misc] mei: fix Unnecessary space after function pointer name

Message-id: <1399041470-21330-74-git-send-email-prarit@redhat.com>
Patchwork-id: 79356
O-Subject: [RHEL7.1 PATCH BZ 1088953 73/77] mei: fix Unnecessary space after function pointer name
Bugzilla: 1088953
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

commit d63b30954cba971377ce51d9df16949572bc1289
Author: Tomas Winkler <tomas.winkler@intel.com>
Date:   Sun Mar 16 14:35:54 2014 +0200

    mei: fix Unnecessary space after function pointer name

    Fix checkpatch warining:
    Unnecessary space after function pointer name

    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Cc: Dean Nelson <dnelson@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/mei_dev.h b/drivers/misc/mei/mei_dev.h
index 24c27ab..108f96d 100644
--- a/drivers/misc/mei/mei_dev.h
+++ b/drivers/misc/mei/mei_dev.h
@@ -243,20 +243,20 @@ struct mei_cl {
  */
 struct mei_hw_ops {
 
- bool (*host_is_ready) (struct mei_device *dev);
+ bool (*host_is_ready)(struct mei_device *dev);
 
- bool (*hw_is_ready) (struct mei_device *dev);
- int (*hw_reset) (struct mei_device *dev, bool enable);
- int  (*hw_start) (struct mei_device *dev);
- void (*hw_config) (struct mei_device *dev);
+ bool (*hw_is_ready)(struct mei_device *dev);
+ int (*hw_reset)(struct mei_device *dev, bool enable);
+ int (*hw_start)(struct mei_device *dev);
+ void (*hw_config)(struct mei_device *dev);
 
- void (*intr_clear) (struct mei_device *dev);
- void (*intr_enable) (struct mei_device *dev);
- void (*intr_disable) (struct mei_device *dev);
+ void (*intr_clear)(struct mei_device *dev);
+ void (*intr_enable)(struct mei_device *dev);
+ void (*intr_disable)(struct mei_device *dev);
 
- int (*hbuf_free_slots) (struct mei_device *dev);
- bool (*hbuf_is_ready) (struct mei_device *dev);
- size_t (*hbuf_max_len) (const struct mei_device *dev);
+ int (*hbuf_free_slots)(struct mei_device *dev);
+ bool (*hbuf_is_ready)(struct mei_device *dev);
+ size_t (*hbuf_max_len)(const struct mei_device *dev);
 
  int (*write)(struct mei_device *dev,
        struct mei_msg_hdr *hdr,
@@ -265,7 +265,7 @@ struct mei_hw_ops {
  int (*rdbuf_full_slots)(struct mei_device *dev);
 
  u32 (*read_hdr)(const struct mei_device *dev);
- int (*read) (struct mei_device *dev,
+ int (*read)(struct mei_device *dev,
        unsigned char *buf, unsigned long len);
 };
 
-- 
1.7.1